mappings/net/minecraft/world/gen/DensityFunctions.mapping

```diff
@@ -0,0 +1,445 @@
+		ARG 0 input
+		ARG 1 minInclusive
+		ARG 3 maxInclusive
+		ARG 5 whenInRange
+		ARG 6 whenOutOfRange
+	METHOD m_nqecyxgz cacheOnce (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function
+	METHOD m_nsnpgtzo bootstrap (Lnet/minecraft/unmapped/C_tqxyjqsk;)Lcom/mojang/serialization/Codec;
+		ARG 0 registry
+	METHOD m_okrngdwc noise (Lnet/minecraft/unmapped/C_cjzoxshv;DD)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 noise
+	METHOD m_ovaexbma add (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function1
+		ARG 1 function2
+	METHOD m_pfuzzaai constant (D)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 value
+	METHOD m_pvonorqa makeCodec (Lcom/mojang/serialization/MapCodec;)Lcom/mojang/serialization/Codec;
+	METHOD m_qjocdzol noise (Lnet/minecraft/unmapped/C_cjzoxshv;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 noise
+	METHOD m_qwccqgbn mappedNoise (Lnet/minecraft/unmapped/C_cjzoxshv;DDDD)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 noise
+	METHOD m_qwpxedjm map (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_mdkmcezi$C_ucbebgzr$C_vgrinicv;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 input
+		ARG 1 type
+	METHOD m_rrapcnud slide (Lnet/minecraft/unmapped/C_ulvgadxi;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 settings
+		ARG 1 input
+	METHOD m_skodlkvx cacheAllInCell (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function
+	METHOD m_sswjlstj mapFromUnitTo (Lnet/minecraft/unmapped/C_kzwapyud;DD)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 value
+	METHOD m_tofdnidr endIslands (J)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 random
+	METHOD m_tykuysov interpolated (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function
+	METHOD m_vgtncips max (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function1
+		ARG 1 function2
+	METHOD m_yhzayeyi register (Lnet/minecraft/unmapped/C_tqxyjqsk;Ljava/lang/String;Lcom/mojang/serialization/Codec;)Lcom/mojang/serialization/Codec;
+		ARG 0 registry
+		ARG 1 id
+		ARG 2 entry
+	METHOD m_zbjmzdyp singleFunctionArgumentCodec (Ljava/util/function/Function;Ljava/util/function/Function;)Lcom/mojang/serialization/Codec;
+	CLASS C_aovxcpjj ClampedGradientY
+		FIELD f_brhvizcz fromY I
+		FIELD f_fxihhpnv fromValue D
+		FIELD f_nncsoykg toValue D
+		FIELD f_onhqlrmw DATA_CODEC Lcom/mojang/serialization/MapCodec;
+		FIELD f_uqleuzjp CODEC Lcom/mojang/serialization/Codec;
+		FIELD f_yydgzbsh toY I
+		METHOD <init> (IIDD)V
+			ARG 1 fromY
+			ARG 2 toY
+			ARG 3 fromValue
+			ARG 5 toValue
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+		METHOD m_czahebwa fromValue ()D
+		METHOD m_khgjowhl fromY ()I
+		METHOD m_mitcetii toY ()I
+		METHOD m_nearmskl toValue ()D
+	CLASS C_dsnmcorv BeardifierOrMarker
```

Comment:
I don't think this lines up with what we have currently - `Beardifier` in moj is.. not called that in QM